### ISSUE TITLE
debugpy extension not working

### ISSUE THREAD

**Issue by mrajancsr** [microsoft/vscode-python-debugger#211]
## Environment data

- debugpy version: 1.8.0
- OS and version: mac OS Ventura
- Python version: 3.11.7 (virtual environment using venv)
- Using VS Code or Visual Studio: VS Code

## Actual behavior
changed the following configuration to debugpy under type
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Python: Current File",
            "type": "debugpy",
            "request": "launch",
            "program": "${file}",
            "console": "integratedTerminal",
            "cwd": "${fileDirname}",
            "env": {
                "PYTHONPATH": "/neptune-ai"
            }
        }
    ]
}
```
My settings.json
```
{
  "autoDocstring.docstringFormat": "numpy",
  "python.languageServer": "Pylance",
  "editor.formatOnSave": true,
  "editor.codeLens": true,
  "python.analysis.autoImportCompletions": true,
  "code-runner.executorMap": {
    "python": "/research/Home/python/neptune-ai/py_env/bin/python3"
  },
  "code-runner.clearPreviousOutput": true,
  "editor.codeActionsOnSave": {
    "source.organizeImports": "explicit"
  },
  "git.ignoreLimitWarning": true,
  "terminal.integrated.env.osx": {
    "PYTHONPATH": "/research/Home/python/neptune-ai"
  },
  "jupyter.notebookFileRoot": "${workspaceFolder}",
  "[python]": {
    "editor.defaultFormatter": "ms-python.black-formatter",
    "editor.formatOnSave": true
  },
  "python.testing.unittestArgs": [
    "-v",
    "-s",
    ".",
    "-p",
    "*test*.py"
  ],
  "python.testing.unittestEnabled": true,
}
```

## Expected  behavior
When hitting dubug in top left it complains that i need to select a python interpreter before i can debug and it stops the debugging process.  But when i change the type back to python, its able to debug as long as i have selected Python: Current File.  Also the top right debug button isn't working under this setting as well.  

## Steps to reproduce:
See above

**Comment by Josverl** [comment]
same/similar issue on Windows 11 with Python 3.11 installed via the store app

- .venv created via vscode 
- running scrips / modules work fine in the .venv 
- debugging keeps asking to select a valid interpreter
- if I `pip install debugpy`  into the .venv and start debugpy manually I can connect 'Remote Attach' to that running process and debugging works fine - but that is a clumsy workaround

**Comment by paulacamargo25** [comment]
Thanks for your bug report, im going to investigate and try to repro de error.

**Comment by mrajancsr** [comment]
> Thanks for your bug report, im going to investigate and try to repro de error.

may i know whats going on?  its already been over 2 weeks.  Causing huge pain in my workflow and productivity at the moment.

**Comment by paulacamargo25** [comment]
This issue will be track in https://github.com/microsoft/vscode-python-debugger/issues/216.

**Comment by zam1kor** [comment]
is this issue fixed ? with version 1.8.1, I still face the same issue :(
